.github/workflows/django.yml

```diff
@@ -1,15 +1,15 @@
 name: Django CI
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+# Execute the workflow with every push
+on: [push]
```

Comment:
Please revert this line back to -
```yml
on:
    push:
        branches: [ master ]
```

Reply:
@hamhaingaurav The idea is to execute the pipeline and the tests with every push to any branch. It will help identify any failing test immediately. You will hit `merge` only if the pipeline executes successfully.
If we restrict it to just the master branch, then I believe the pipeline will execute only if the changes are pushed to master.

Identifying issues early in the workflow is a good practice. What do you say?

Reply:
@hamhaingaurav Did you get a chance to read my comment above?

Reply:
@anoop1409 , I feel you have not read my comment, so I am posting it again,

"That can be proven good for other projects, as this project is not a large one, we can happily manage on push to master only. Also, please consider GitHub limits on Free plan, GitHub provides only 2000 action minutes/month, and my account have multiple projects, some of them are on private repo, it will tear apart my account limits."